src/components/Tables/Table.tsx

```diff
@@ -311,35 +292,37 @@ const Table: React.FC<Props> = ({
         });
     }
 
-    const onFilter = (event: React.ChangeEvent<HTMLInputElement>, filterField: string): void => {
-        setFilterText({ ...filterText, [filterField]: event.target.value });
-    };
+    const rows = data.map((data, index) => ({ rowId: index, data }));
 
-    const handleClear = (): void => {
-        if (filterText) {
-            setFilterText({});
-        }
+    const shouldFilterRow = (row: Row<Data>): boolean => {
+        return filters.every((filter) => !filter.shouldFilter(row.data, filter.state));
+        // return true;
```

Comment:
is this comment needed? I have no clue what it is referring to so if so it needs more description

Reply:
nope, that was for debugging I'll get rid of it